DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Operations/Send/RootObjectBuilder.cs

```diff
@@ -58,47 +59,50 @@ public RootObjectBuilder(IUnitOfWorkFactory unitOfWorkFactory)
 
     Collection rootObjectCollection = new(); //TODO: Collections
 
-    foreach (string uri in uriList)
+    QueuedTask.Run(() =>
```

Comment:
Is this `QueuedTask` required. I see we have one already in the `SendOperation`

If and when we were to move `SendOperation` to a shared class, this is where any syncing to main threads will occur. I'd be curious if we may still require more syncing going on in the converters...

Reply:
Indeed! I suppose, we remove it from here and leave a POC comment in SendOperation? To check whether move it down the hierarchy